normalize_repos/labels.json

```diff
@@ -0,0 +1,163 @@
+{
+  "groups": [
+    {
+      "name": "priority",
+      "labels": [
+        {
+          "name": "critical",
+          "color": "UNFAVOURABLE",
+          "description": "Issue breaks production and needs fixing asap",
+          "emoji": "\uD83D\uDFE5"
+        },
+        {
+          "name": "high",
+          "color": "NEGATIVE",
+          "description": "Issue stalls works on the project or its dependents",
+          "emoji": "\uD83D\uDFE7"
+        },
+        {
+          "name": "medium",
+          "color": "NEUTRAL",
+          "description": "Issue is not up actively harmful but that can escalate",
+          "emoji": "\uD83D\uDFE8"
+        },
+        {
+          "name": "low",
+          "color": "POSITIVE",
+          "description": "Issue doesn't hurt anyone",
```

Comment:
Issues can be for new features, which doesn't have anything to do with harming users by their absence. I would just say that there's no hurry in resolving this issue.